### DISCUSSION TITLE
FL 1.9.5. Modify translation of the description of the Plugin "System Commands". Proposal

### DISCUSSION THREAD

**Discussion by dracp** [Flow-Launcher/Flow.Launcher#1442] [Ideas]
Considering that the text describes (explains, details,...) the function performed by the command in question and not expressly the action to be executed, it seems more convenient to translate the descriptions with a verbal expression in the formal third person (él, usted) instead of using the infinitive of the verb, more appropriate to express an action in itself.

As the descriptions of the different "Plugins" have been translated.

NOTE
The proposed translation has been introduced in Crowdin

![221003 FL 1 9 5_Complementos_Comandos del Sistema_Descripción_3](https://user-images.githubusercontent.com/102802425/193613145-8544b2aa-e5b8-460d-b7d4-84506da6945d.png)

**Comment by dracp** [comment]
In FL 1.10.0, the original text is still the same. It has not been changed.

**Reply by taooceros** [reply]
Has that been changed? I have merged the crowdin branch recently.

**Comment by dracp** [comment]
In Crowdin "“Español (original) > Todas las cadenas” (All strings), no new string to translate into English is shown.
![221007 Crowdin_Todo traducido](https://user-images.githubusercontent.com/102802425/194628913-6c13baa0-4279-4a94-9da0-1320a5dc7e94.png)

**Reply by taooceros** [reply]
Sorry I didn't get the idea. Do you mean that you cannot find the strings for translation? Maybe some of the option is not yet in the resource.

**Comment by dracp** [comment]
I always search in **“All strings”** with the option **“All, without translating first”**. And at the moment I can't find any string to translate.

In any case, I have already incorporated in Crowdin the translation proposed in the image above.

![221007 Crowdin_Descripción de Complementos](https://user-images.githubusercontent.com/102802425/194635583-9ddc8a9b-6bc3-4d16-8a75-43a3b05fae95.png)

**Reply by taooceros** [reply]
Thanks so much! We are a bit slow in this, since there's are too much ongoing things that hasn't been finished. But i believe we will add all these strings before releasing.

**Reply by dracp** [reply]
> Thanks so much! We are a bit slow in this, since there's are too much ongoing things that hasn't been finished. But i believe we will add all these strings before releasing.

I understand the great effort that must be involved in developing the application.
I will continue to indicate those aspects of the translation that have not been included in the application itself.
Waiting, depending on your needs are modified.
Greetings.

**Comment by dracp** [comment]
OK. In the message "dracp 4 days ago", it was already indicated in the NOTE.

**NOTE
The proposed translation has been introduced in Crowdin**

**Comment by dracp** [comment]
**Revised FL 1.10.0, installed on October 08**. Settings/System Commands.
Everything has been translated except for "Hotkey", "Open recycle bin" and "Tweak Flow Launcher's settings".
"Hotkey" and "Open recycle bin", already translated in Crowdin as "Atajos de teclado" and "Abre la papelera de reciclaje".
"Tweak Flow Launcher's settings", has been recently translated as "Ajusta...". (Sorry).

![221009 FL 1 10 0 10 08 22_Comandos del sistema_Traducido_con excepciones](https://user-images.githubusercontent.com/102802425/194753210-c0480608-73d5-4711-a5a3-6e1463624489.png)

**Reply by taooceros** [reply]
Thank you so much for the detailed report! I will check recently (but probably I couldn't do it that fast because of school work).

**Reply by taooceros** [reply]
I think these new translations are included in the translation pipeline, but hasn't been merged.

See https://github.com/Flow-Launcher/Flow.Launcher/pull/1450/files

**Reply by dracp** [reply]
> I think these new translations are included in the translation pipeline, but hasn't been merged.
> 
> See https://github.com/Flow-Launcher/Flow.Launcher/pull/1450/files

I have seen that all three translations have already been included in "New Crowdin updates #1450".

**Reply by taooceros** [reply]
I will merge them recently.

**Reply by dracp** [reply]
> I will merge them recently.

I'm checking the strings associated with "Context: ...filefolder....".
There may be a problem in the Spanish translation due to the different genders of the words "carpeta". and "archivo". "La carpeta" (The folder) is feminine and "El archivo" (The file) is masculine.
The subtitles of the "Actions with files" sheet, as well as the pop-up windows of acceptance or not of the action and the notification cards (Toast), are not displayed correctly in Spanish due to the different gender of "Carpeta" and "Archivo".
I will send a proposal as soon as I have it available.

**Reply by taooceros** [reply]
Thank you so much!

**Reply by taooceros** [reply]
@dracp once you proposed all the change, and these change is reflected in the Crowdin Update pr, please @me so we can merge that in. Thank you so much!

**Reply by dracp** [reply]
Revised FL 1.10.0 (October 12)
Not modified.

**Reply by dracp** [reply]
> @dracp once you proposed all the change, and these change is reflected in the Crowdin Update pr, please @me so we can merge that in. Thank you so much!

I don't know exactly what you are asking for. Do I need to modify anything else?

**Reply by taooceros** [reply]
No no nothing. I just want to merge all in together. In case anything delayed into the next merge schedule.

**Comment by dracp** [comment]
In relation to the matter: 
"I'm checking the strings associated with "Context: ...filefolder....". (3 days ago) 
See:
https://github.com/Flow-Launcher/Flow.Launcher/discussions/1475#discussion-4473861
